numba/parfors/parfor_lowering.py

```diff
@@ -116,55 +118,54 @@ def _lower_parfor_parallel(lowerer, parfor):
             # Reduction array is created and initialized to the initial reduction value.
 
             # First create a var for the numpy empty ufunc.
-            empty_func = ir.Var(scope, mk_unique_var("empty_func"), loc)
-            ff_fnty = get_np_ufunc_typ(np.empty)
-            ff_sig = typingctx.resolve_function_type(ff_fnty,
-                                            (types.UniTuple(types.intp, redarrdim),
-                                             types.DType(reddtype)), {})
-            typemap[empty_func.name] = ff_fnty
-            empty_assign = ir.Assign(ir.Global("empty", np.empty, loc=loc), empty_func, loc)
-            lowerer.lower_inst(empty_assign)
+            empty_bound_node = pfbdr.bind_global_function(
```

Comment:
```suggestion
            glbl_np_empty = pfbdr.bind_global_function(
```
May be use a convention in naming vars based on what their IR type is?